process/block/sovereign/outgoingOperations.go

```diff
@@ -91,17 +96,24 @@ func (op *outgoingOperations) CreateOutgoingTxsData(logs []*data.LogData) [][]by
 		return make([][]byte, 0)
 	}
 
-	txData := []byte(bridgeOpPrefix + "@" + fmt.Sprintf("%d", op.roundHandler.Index()))
-	for _, ev := range outgoingEvents {
-		txData = append(txData, byte('@'))
-		txData = append(txData, createSCRData(ev.GetTopics())...)
-		txData = append(txData, byte('@'))
-		txData = append(txData, ev.GetData()...)
+	txsData := make([][]byte, 0)
+	for i, event := range outgoingEvents {
+		operation, err := op.getOperationData(event)
+		if err != nil {
```

Comment:
can this operation be just skipped with an log.Error? Is there no check for consecutive nonces or something?

Reply:
added error

Reply:
ok